src/api/app/jobs/stage_requests_job.rb

```diff
@@ -0,0 +1,28 @@
+class StageRequestsJob < ApplicationJob
+  def perform(staging_project, request_ids, user)
+    unassigned = []
+    User.current = user
+    requests = BsRequest.find(request_ids)
+    requests.each do |request|
+      action = request.bs_request_actions.first # we only care about the first one
+      if action.is_submit?
+        BranchPackage.new(
+          target_project: staging_project.name,
+          project: action.source_project,
```

Comment:
I think we should add target_package here as well, I forgot to add it :thinking: But I guess it makes sense...

```
target_package: action.target_package,
```